bot.py

```diff
@@ -0,0 +1,214 @@
+import os
+import re
+import sys
+from datetime import datetime
+import time
+
+import discord
+import dotenv
+import redis
+from discord.ext import commands
+from discord_slash import SlashCommand
+from discord_slash.utils.manage_commands import create_option, create_choice
+from dotenv import dotenv_values
+
+r = redis.Redis(host=dotenv_values("config.env")["HOSTNAME"], port=dotenv_values("config.env")["PORT"], db=dotenv_values("config.env")["DB"])
```

Comment:
1. Caps for CONSTS
2. Create Redis module that will extract setting from the config.env and import it
In the config file - split configs to SQL, Redis and etc

Reply:
Created other projects for different database management, might create one mega bot later containing all of the other databases frameworks